text/ps-signatures/README.md

```diff
@@ -83,6 +83,20 @@ There are 3 mainly entities in anonymous credentials, the *issuer* which signs a
         ) -> Result<Self, PSError>
     }
     ```
+- To generate the challenge, the prover will need to get bytes for the pre-challenge phase. This can be done by using `PoKOfSignature::to_bytes` method.
+- For the verifier to generate the challenge, it will get bytes for the pre-challenge phase by calling `PoKOfSignatureProof::get_bytes_for_challenge`. `revealed_msg_indices` corresponds to the messages revealed to the verifier since they are not included in the proof of knowledge protocol.
```

Comment:
The Verifier generates the challenge by retrieving all relavant bytes for the pre-challenge phase by calling ...

Reply:
Done